src/main/java/org/assertj/core/api/AbstractFileAssert.java

```diff
@@ -691,6 +692,15 @@ public SELF hasName(String expected) {
     return myself;
   }
 
+  /**
+   * Alias for {@link #hasName(String)}.
+   *
+   * This method can be used to easily switch between {@link File} and {@link Path} assertions.
```

Comment:
I think the correct way (judging by surrounding code) is to use `@code` instead of `@link`.
Imho like 698 is not really needed - like 696 explains everything.

Reply:
Tired to use the javadoc similar to [`assertThatNullPointerException` javadoc](https://github.com/assertj/assertj-core/blob/main/src/main/java/org/assertj/core/api/Assertions.java#L1416-L1422)

Removed line 698.